packages/blockchain-wallet-v4-frontend/src/components/CircularProgressBar/index.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import { buildStyles, CircularProgressbarWithChildren } from 'react-circular-progressbar'
+
+import 'react-circular-progressbar/dist/styles.css'
+
+const ProgressCircle = ({ children, percentage, strokeWidth }) => {
+  return (
+    <CircularProgressbarWithChildren
+      styles={buildStyles({
+        backgroundColor: '#0C6CF2',
```

Comment:
possible to get these from the theme?

Reply:
well I might need direct import of color from theme, no other way to pass it over since this is not styled component :/

Reply:
yeah you are right there is a hook useTheme in styled components which helps to use theme outside of styled components which is a case over here ;)